ansible/roles/stanchion/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Fetch Stanchion install script
+  command: curl -o {{ stanchion_install_script }} {{ stanchion_script_url }}
+
+- name: Run Stanchion install script
+  command: bash {{ stanchion_install_script }}
```

Comment:
rename tasks to something like 'Fetch Stanchion repo install script' and 'Install Stanchion repo'